controllers/controllers/workloads/cfbuild_controller_test.go

```diff
@@ -170,6 +172,14 @@ var _ = Describe("CFBuildReconciler", func() {
 
 				Expect(actualWorkload.Spec.Env).To(Equal(buildEnv))
 			})
+
+			It("sets the build reconciler from the controller config", func() {
+				Expect(fakeClient.CreateCallCount()).To(Equal(1), "fakeClient Create was not called 1 time")
+				_, obj, _ := fakeClient.CreateArgsForCall(0)
```

Comment:
I thought we'd agreed not to add these optional assertion descriptions except for truly confusing/unclear circumstances. They mostly just clutter the code.

Reply:
IMHO, it is more about the clarity of the test output than the code. Any time you see a test failure where Ginkgo expected 0 to equal 1 (or some variation of this), it can be extremely confusing as to what is failing without digging into the code.

I think that at least part of the problem is that we are not consistent about adding these helper descriptions.